sampleapp/src/main/java/com/zendesk/connect/sampleapp/MainActivity.kt

```diff
@@ -59,15 +34,64 @@ class MainActivity : AppCompatActivity() {
             trackEvent(sampleEvent)
         }
 
+        register_button.setOnClickListener {
+            registerForPush()
+        }
+
+        disable_button.setOnClickListener {
+            disablePushNotifications()
+        }
+
+        logout_button.setOnClickListener {
+            logout()
+        }
+
     }
 
+    /**
+     * Identifies a user within Outbound. Should be used when a user logs in or their
+     * identifying information is updated.
+     *
+     * @param user: The user to be identified
+     */
     private fun identifyUser(user: User) {
         Log.d(LOG_TAG, "Identifying User")
         Outbound.identify(user)
     }
 
+    /**
+     * Tracks the given event
+     *
+     * @param event: The event to be tracked
+     */
     private fun trackEvent(event: Event) {
         Log.d(LOG_TAG, "Tracking event")
         Outbound.track(event)
     }
+
+    /**
+     * Registers the identified user for push notifications. The user should be
```

Comment:
I'd clarify the `should be` part of this. Does `should be` mean that the back-end should do this when it identifies a user? Or is it more a case of `You should register the user as soon as/after they are identified.`
If integrators are going to be reading this, it's important to make it clear what they do/don't need to do.

Reply:
So this is just some weird cross over between methods internally. When you call identify it will also register that user for push notifications, but if you call register first then it will identify that user as anonymous and register them for push.

I'll reword the comment to be clearer.